runtime/util/thrinfo.c

```diff
@@ -287,7 +287,17 @@ getVMThreadStateHelper(J9VMThread *targetThread,
 						 */
 					}
 				}
-	
+
+#if JAVA_SPEC_VERSION >= 19
+				/* for thread with a virtual thread mount that is waiting on a lock,
+				 * we should return the virtualthreadobject as lock object/owner */
```

Comment:
comment formatted and reworded; copy as-is; unless there is a mistake.
```suggestion
				/* For a J9VMThread with a mounted virtual thread that is waiting on a
				 * lock, the virtual thread object should be returned for the lock object
				 * and the corresponding J9VMThread should be returned for the lock
				 * owner.
				 */
```

Reply:
Also, a reason should be provided in the comment to highlight `Why this behaviour is needed?`

Reply:
This behaviour is to match the RI's JCL behaviour for ThreadMXBean, there is no explicit documentation for this. The [ThreadMXBean Doc](https://docs.oracle.com/en/java/javase/19/docs/api/java.management/java/lang/management/ThreadMXBean.html) specified that VirtualThread are not supported. Hence any lock related to VirtualThread has not been defined